Tests/SonarScanner.MSBuild.Common.Test/OperatingSystemProviderTests.cs

```diff
@@ -79,7 +88,65 @@ public void IsAlpine_OnInvalidAccess_ReturnsFalse()
         fileWrapper.When(x => x.ReadAllText("/etc/os-release")).Do(_ => throw exception);
         var sut = new OperatingSystemProvider(fileWrapper, logger);
 
-        sut.IsAlpine().Should().Be(false);
+        sut.OperatingSystem().Should().Be(PlatformOS.Linux);
         logger.Received(1).LogWarning("Cannot detect the operating system. {0}", exception.Message);
     }
+
+    [TestMethod]
+    [TestCategory(TestCategories.NoLinux)]
+    public void OperatingSystem_WindowsAndMacOS_AreNotAlpine() =>
+        new OperatingSystemProvider(FileWrapperForAlpine(), Substitute.For<ILogger>()).OperatingSystem().Should().NotBe(PlatformOS.Alpine);
+
+    [TestMethod]
+    [TestCategory(TestCategories.NoLinux)]
+    [TestCategory(TestCategories.NoMacOS)]
+    public void IsUnix_Windows() =>
```

Comment:
The test name is correct and follows our usual convention. Although in this particular case, this could make it slightly clearer - applies for all UTs around here
```suggestion
    public void IsUnix_OnWindows() =>
```